src/redis_ai_objects/script_struct.h

```diff
@@ -30,19 +30,3 @@ typedef struct RAI_Script {
     AI_dict *functionData; // A <String, TorchScriptFunctionArgumentType*> dict to map between
                            // function name, and its schema.
 } RAI_Script;
-
```

Comment:
Maybe this whole file is redundant, and this structs should move to `script.h`? WDYT?

Reply:
let's discuss it and decide
different PR scope